guide/english/typescript/classes/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Classes
+---
+
+# Classes
+
+`Object Oriented Programming` is worthwhile where tons of code are written, by structuring the code it makes vivid to understand.
```

Comment:
Please re-phrase these lines so they are easier to understand.
> `Object Oriented Programming` is worthwhile where tons of code are written, by structuring the code it makes vivid to understand.
Class is one of the paradigm of `OOP`.
TypeScript class reserved `class` keyword to create a class. The syntax is given below - 

Maybe something like this -

```suggestion
Object Oriented Programming is an important programming principle, especially where a lot of code is written. By structuring the code in classes and objects, it makes the code easier to understand.
```

Same with this line -
> TypeScript class reserved `class` keyword to create a class. The syntax is given below -